### PR TITLE
fix(select): show arrow in the dropdown

### PR DESCRIPTION
**Pull request by Befkadu1**
### Browsers tested:

Windows:
- [ ] Chrome
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/select/select.scss [hunk 1]
```diff
@@ -91,7 +91,11 @@ $mdc-select-bottom-line-hover-color: $lime-text-field-bottom-line-color;
             background-color: var(--mdc-theme-primary);
         }
 
-        .limel-select__dropdown-icon {
+        .mdc-select__dropdown-icon {
+            // width: pxToRem(28);
+            // margin-left: pxToRem(8);
+            // margin-top: pxToRem(32);
+            // margin-bottom: pxToRem(8);
```
Comment (jgroth) [inline src/components/select/select.scss]: We should remove this commented code :slightly_smiling_face:

Comment (Befkadu1) [reply src/components/select/select.scss]: Removed

**Comment by adrianschmidt** [comment]
Great to see you contributing already Bef! ❤️

**Comment by Befkadu1** [comment]
Thanks Adrian.

**Pull request merged**